src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,49 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final int SECOND_ELEMENT = 1;
+    private static final String TXT_SUPPLY = "supply";
+    private static final String TXT_BUY = "buy";
+    private static final String TXT_RESULT = "result";
+    private static final String TXT_SEPARATOR = ",";
+
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
The method `getStatistic` is doing too much. It's reading from a file, processing the data, and then writing to another file. According to the 'Single Responsibility Principle', a method should do one thing and do it well. Consider breaking this method into smaller ones, each with a single responsibility.